src/features/home/ui/StackSelectButton.tsx

```diff
@@ -0,0 +1,16 @@
+import * as S from './stackSelectButton.style';
+
+interface ButtonProps {
+  children: string;
+  src?: string;
+  onClick?: () => void;
+}
+
+export const StackSelectButton = ({ children, src, onClick }: ButtonProps) => {
+  return (
+    <S.Container onClick={onClick}>
+      <S.StackImg src={src} alt="stack-button" />
```

Comment:
만약에 화면에 보여줄 문자열을 text라는 prop으로 받는다는 가정하에 alt는 이런식으로 표현해주면 좋을 것 같아요~
```suggestion
      <S.StackImg src={src} alt={`${text} image`} />
```

Reply:
https://github.com/GrebbMe/grabbme-fe/pull/58/commits/72a0f01e02968da1df45ab4b9b92f6da6ee3fcca 에서 변경해주었습니다!